clients/cli/src/main/java/org/apache/gravitino/cli/outputs/PlainFormat.java

```diff
@@ -80,4 +83,11 @@ public void output(Catalog[] catalogs) {
       System.out.println(all);
     }
   }
+
+  static final class SchemaPlainFormat implements OutputFormat<Schema> {
+    @Override
+    public void output(Schema schema) {
+      System.out.println(schema.name() + "," + schema.comment());
+    }
```

Comment:
This is missing the null check of the previous code. As per more recent PRs, it should also output "No schemas exist." if there are none.